prow/plugins.yaml

```diff
@@ -503,12 +503,18 @@ plugins:
   - override
 
   kubernetes/release:
+  - blockade
```

Comment:
The `blockade` plugin shouldn't be enabled on `kubernetes/release` or `kubernetes/sig-release` without configuring any blockades: https://github.com/kubernetes/test-infra/blob/e96bdfa7a70931b72bebe25aca2a68de13ab18cd/prow/plugins.yaml#L136-L159

Reply:
@cjwagner -- the lack of current blockades doesn't preempt enabling the plug-in though, right?

I wanted it enabled in advance, so if/when we add blockades there's only one change to make.

Reply:
Not strictly, but enabling the plugin without any blockades will still waste an API token because we have to check if the blocker label needs to be removed when no blockades are configured. https://github.com/kubernetes/test-infra/blob/894b7437551f33e6000d598b28f9ff865cd4a72a/prow/plugins/blockade/blockade.go#L140
This is true for a number of plugins so typically we avoid enabling plugins in places they won't be used (e.g. enabling blockade globally for the `kubernetes` org instead of individual repos).

FWIW the plugin can be enabled at the same time you configure the blockades so it doesn't require an additional PR if that is what you mean.

This won't be prohibitively expensive though so we can just go ahead and merge this. (After all the plugin will use even more tokens when blockades are configured for the repos and it actually has to do work.) This is more of general best practice to avoid wasting API tokens.
Feel free to `/hold cancel`

Reply:
Really appreciate the additional context, @cjwagner! I removed the blockade plugin here and opened #12266 for someone to add some notes around what you mentioned. :)

Reply:
Definitely prefer to do things the right way, as I usually copy/paste other repos plugin configs, if I'm working on a new one.